ip/mp4.c

```diff
@@ -26,6 +26,9 @@
 #endif
 #include "../comment.h"
 #include "aac.h"
+#include "misc.h"
```

Comment:
Shouldn't this be:
```
#include "../xstrjoin.h"
#include "../misc.h"
#include "../file.h"
```
?
I get include errors when compiling as is.

Reply:
Yes you're right, I'll fix this.